### PR TITLE
Update Docs: Intro “start-with-a-template”

### PR DESCRIPTION
**Pull request by wernerglinka**
Automatically generated by Netlify CMS

### CODE REVIEW

File: website/content/docs/start-with-a-template.md [hunk 1]
```diff
@@ -55,6 +55,13 @@ You can add Netlify CMS [to an existing site](/docs/add-to-your-site/), but the
         <h4>Nuxt.js Boilerplate</h4>
         <p><a href="https://app.netlify.com/start/deploy?repository=https://github.com/Knogobert/ntn-boilerplate&amp;stack=cms"><img src="https://www.netlify.com/img/deploy/button.svg" alt="Deploy to Netlify" /></a></p>
     </div>
+    <div style="flex-basis: 33%">
+        <div style="padding: 0 30%; height: 100px; display: flex; justify-content: center;">
+            <img style="display: flex" src="/img/metalsmith.png"/>
```
Comment (erezrokah) [inline website/content/docs/start-with-a-template.md:60]: @wernerglinka do you mind adding the logo under `website/static/img`?

The image is broken without it, see https://deploy-preview-6324--netlify-cms-www.netlify.app/docs/start-with-a-template/

image

**Review by erezrokah — CHANGES_REQUESTED** [review]
Thanks @wernerglinka, very cool 🎉 

Looks good pending one comment.

File: website/content/docs/start-with-a-template.md (hunk 1 above)
Comment (wernerglinka) [reply website/content/docs/start-with-a-template.md:60]: metalsmith

**Comment by erezrokah** [comment]
Closing in favor of https://github.com/netlify/netlify-cms/pull/6327

**Pull request closed without merging**